modules/mono/glue/GodotSharp/GodotSharp/Core/Colors.cs

```diff
@@ -171,12 +172,14 @@ public static class Colors
         public static Color Blue { get { return namedColors["blue"]; } }
         public static Color BlueViolet { get { return namedColors["blueviolet"]; } }
         public static Color Brown { get { return namedColors["brown"]; } }
-        public static Color BurlyWood { get { return namedColors["burlywood"]; } }
+        public static Color BurlyWood { get { return namedColors["burlywood"]; } } // for compatibility
+        public static Color Burlywood { get { return namedColors["burlywood"]; } }
```

Comment:
It should be kept as `BurlyWood`. The two words stand fine on their own, and "Burlywood" isn't a very common word, but most importantly, it matches the X11 and .NET color names.

Reply:
Wikipedia is incorrect? Welp, not surprised.